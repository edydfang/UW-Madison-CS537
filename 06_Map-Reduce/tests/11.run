./wordcount tests/1.txt tests/2.txt 2 1 1 > tests-out/11.mid; python3 wordcount_parser.py --input-file tests-out/11.mid