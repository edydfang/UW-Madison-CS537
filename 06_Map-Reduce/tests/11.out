3344556677 1
3456346 1
34565363 1
523654 1
653465 1
75687785 1
